py_retro/game_info_reader.py

```diff
@@ -0,0 +1,15 @@
+import ctypes
+module_name = "get_game_info"
+
+class GameInfoReader:
+    def get_gambatte_game_info(self, data):
+        return {
+            'name': ctypes.string_at(data[0x134:0x143])
+        }
+
+    def get_info(self, romData, coreName):
+        getInfoForCore = getattr(self, "get_{}_game_info".format(coreName.lower()))
```

Comment:
note that some cores have spaces in their names, i think a good approach would be a `get_gbc_game_info` and a `{'gambatte': 'gbc', 'TGB Dual': 'gbc', 'bsnes': 'snes', ...}`